conda/recipes/libcuspatial/meta.yaml

```
@@ -86,12 +106,24 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
     requirements:
       build:
         - cmake {{ cmake_version }}
+      host:
+        - cuda-version ={{ cuda_version }}
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        {% else %}
+        - cuda-cudart-dev
+        {% endif %}
```

If we add the `compiler`s above, the `cudatoolkit` portion will be unneeded

```suggestion
        {% if cuda_major != "11" %}
        - cuda-cudart-dev
        {% endif %}
```

---

Resolving, see https://github.com/rapidsai/cuspatial/pull/1044#discussion_r1248296401